app/src/components/buttons/select-token-or-network-button.tsx

```diff
@@ -75,6 +86,17 @@ export const SelectTokenOrNetworkButton = (props: Props) => {
           {text ?? getLocale('braveSwapSelectToken')}
         </Text>
       </Row>
+      {gasEstimate && (
```

Comment:
Should we update this to:
```suggestion
      {networkFeeFiatValue && (
```
?

Reply:
Updated!